plugins/processors/awsapplicationsignals/internal/normalizer/attributesnormalizer.go

```diff
@@ -60,9 +62,9 @@ var attributesRenamingForTrace = map[string]string{
 	attr.AWSRemoteTarget: attr.AWSRemoteResourceIdentifier,
 }
 
-var copyMapForMetric = map[string]string{
+var resourceAttributesToMetric = map[string]string{
```

Comment:
nit: The current name makes it sound like the resource attributes are being converted into metrics whereas I believe it's a copy and rename.
```suggestion
var resourceToMetricAttributes = map[string]string{
```